examples/mbedTLS/CMakeLists.txt

```diff
@@ -0,0 +1,72 @@
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-mbedtls)
+
+hunter_add_package(mbedtls)
+find_library(
```

Comment:
It's better to add CMake code to generate `mbedtlsConfig.cmake` so users can do `find_package(mbedtls CONFIG REQUIRED)` instead.